### PR TITLE
Add iFAction Game Maker Engine

### PR DESCRIPTION
**Pull request by MichaelHinrichs**
### SteamDB app page links to a few games using this
https://store.steampowered.com/app/2847250
https://store.steampowered.com/app/2281410
https://store.steampowered.com/app/1195270
https://store.steampowered.com/app/1593790

### Brief explanation of the change
Find the files from iFAction Game Maker.

### CODE REVIEW

**Comment by SteamTracker** [comment]
After running changed rules in this pull request *(this bot only tests rules.ini changes)*:

Changed technologies: **Engine.iFAction\_Game\_Maker**

These apps now match: **(38)** [856060](https://steamdb.info/app/856060/depots/) [1195270](https://steamdb.info/app/1195270/depots/) [1346740](https://steamdb.info/app/1346740/depots/) [1371280](https://steamdb.info/app/1371280/depots/) [1380320](https://steamdb.info/app/1380320/depots/) [1381270](https://steamdb.info/app/1381270/depots/) [1388690](https://steamdb.info/app/1388690/depots/) [1397700](https://steamdb.info/app/1397700/depots/) [1401940](https://steamdb.info/app/1401940/depots/) [1405880](https://steamdb.info/app/1405880/depots/) [1442040](https://steamdb.info/app/1442040/depots/) [1493320](https://steamdb.info/app/1493320/depots/) [1522940](https://steamdb.info/app/1522940/depots/) [1568060](https://steamdb.info/app/1568060/depots/) [1593790](https://steamdb.info/app/1593790/depots/) [1606590](https://steamdb.info/app/1606590/depots/) [1633460](https://steamdb.info/app/1633460/depots/) [1751230](https://steamdb.info/app/1751230/depots/) [1894150](https://steamdb.info/app/1894150/depots/) [1936480](https://steamdb.info/app/1936480/depots/) [2012830](https://steamdb.info/app/2012830/depots/) [2078830](https://steamdb.info/app/2078830/depots/) [2081240](https://steamdb.info/app/2081240/depots/) [2084020](https://steamdb.info/app/2084020/depots/) [2094760](https://steamdb.info/app/2094760/depots/) [2102880](https://steamdb.info/app/2102880/depots/) [2136870](https://steamdb.info/app/2136870/depots/) [2228940](https://steamdb.info/app/2228940/depots/) [2269420](https://steamdb.info/app/2269420/depots/) [2281410](https://steamdb.info/app/2281410/depots/) [2320030](https://steamdb.info/app/2320030/depots/) [2343730](https://steamdb.info/app/2343730/depots/) [2446080](https://steamdb.info/app/2446080/depots/) [2483970](https://steamdb.info/app/2483970/depots/) [2687080](https://steamdb.info/app/2687080/depots/) [2767610](https://steamdb.info/app/2767610/depots/) [2847250](https://steamdb.info/app/2847250/depots/) [2875650](https://steamdb.info/app/2875650/depots/)

Please confirm that all these changed apps are correct.

File: descriptions/Engine.iFAction_Game_Maker.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[**iFAction**](http://www.cedong.com.cn/index.html) is a visualized game making software.
```
Comment (Lifeismana) [inline descriptions/Engine.iFAction_Game_Maker.md:1]: needs a website in english

File: rules.ini [hunk 2]
```diff
@@ -111,6 +111,8 @@ idTech[] = \.streamed$
 idTech[] = \.mega2$
 idTech[] = \.texdb$
 idTech[] = \.streamdb$
+iFAction_Game_Maker[] = \.if(?:Map|sharp|set|app|res|ui)$
+iFAction_Game_Maker[] = (?:^|/)iF(?:Con|ActionGame\.dll)$
```
Comment (Lifeismana) [inline rules.ini:115]:
```suggestion
iFAction_Game_Maker = (?:^|/)iFCon$
```

**Review by Lifeismana — CHANGES_REQUESTED** [review]

File: descriptions/Engine.iFAction_Game_Maker.md (hunk 1 above)
Comment (MichaelHinrichs) [reply descriptions/Engine.iFAction_Game_Maker.md:1]: There isn't one.

**Comment by SteamDB-Tracker** [comment]
After running changed rules in this pull request *(this bot only tests rules.ini changes)*:

Changed technologies: **Engine.iFAction\_Game\_Maker**

These apps now match: **(39)** [856060](https://steamdb.info/app/856060/depots/) [1346740](https://steamdb.info/app/1346740/depots/) [1371280](https://steamdb.info/app/1371280/depots/) [1380320](https://steamdb.info/app/1380320/depots/) [1381270](https://steamdb.info/app/1381270/depots/) [1388690](https://steamdb.info/app/1388690/depots/) [1397700](https://steamdb.info/app/1397700/depots/) [1401940](https://steamdb.info/app/1401940/depots/) [1405880](https://steamdb.info/app/1405880/depots/) [1442040](https://steamdb.info/app/1442040/depots/) [1493320](https://steamdb.info/app/1493320/depots/) [1522940](https://steamdb.info/app/1522940/depots/) [1568060](https://steamdb.info/app/1568060/depots/) [1606590](https://steamdb.info/app/1606590/depots/) [1633460](https://steamdb.info/app/1633460/depots/) [1751230](https://steamdb.info/app/1751230/depots/) [1894150](https://steamdb.info/app/1894150/depots/) [1936480](https://steamdb.info/app/1936480/depots/) [2012830](https://steamdb.info/app/2012830/depots/) [2081240](https://steamdb.info/app/2081240/depots/) [2084020](https://steamdb.info/app/2084020/depots/) [2094760](https://steamdb.info/app/2094760/depots/) [2102880](https://steamdb.info/app/2102880/depots/) [2136870](https://steamdb.info/app/2136870/depots/) [2228940](https://steamdb.info/app/2228940/depots/) [2269420](https://steamdb.info/app/2269420/depots/) [2281410](https://steamdb.info/app/2281410/depots/) [2320030](https://steamdb.info/app/2320030/depots/) [2343730](https://steamdb.info/app/2343730/depots/) [2446080](https://steamdb.info/app/2446080/depots/) [2483970](https://steamdb.info/app/2483970/depots/) [2664850](https://steamdb.info/app/2664850/depots/) [2687080](https://steamdb.info/app/2687080/depots/) [2767610](https://steamdb.info/app/2767610/depots/) [2847250](https://steamdb.info/app/2847250/depots/) [2875650](https://steamdb.info/app/2875650/depots/) [2989460](https://steamdb.info/app/2989460/depots/) [3123630](https://steamdb.info/app/3123630/depots/) [3140700](https://steamdb.info/app/3140700/depots/)

Please confirm that all these changed apps are correct.

**Comment by Lifeismana** [comment]
merged in 9904454fbe984aec2277a7b33f3e155d9c689c8c

**Pull request closed without merging**